.github/workflows/cli-release.yaml

```diff
@@ -0,0 +1,68 @@
+# This defines a workflow to build and release a new version of the aptos CLI.
+# In order to trigger it go to the Actions Tab of the Repo, click "Release CLI" and then "Run Workflow".
```

Comment:
does "Release CLI" need to be created manually after this workflow config lands?

Reply:
it will be created automatically once this workflow lands in Main. Github's UI only displays these workflow dispatch triggered (=manually triggered) Workflows once they're in main, but once they are you can also trigger a different version of the workflow that is only contained in another branch.